packages/site/src/data/graphql/libraries.ts

```diff
@@ -1,26 +1,490 @@
 import { Library } from "@framework/system/src/models/library"
 
 export const libraryTags = [
-	"state management",
-	"internationalization",
-	"framework",
-	"documentation",
-	"styling",
-	"component",
-	"logging",
-	"RxJS",
-	"performance tools",
-	"redux",
-	"firebase",
-	"data fetching",
-	"GraphQL",
-	"Utility-First CSS",
-	"CSS in JS",
-	"component library",
-	"bootstrap",
-	"forms",
-	"reactive forms",
-	"testing",
+	"Servers",
+	"Clients",
+	"Schema Builders",
+	"Combining Schemas",
+	"Misc",
 ] as const
 
-export const libraries: Library<typeof libraryTags[number]>[] = []
+export const libraries: Library<typeof libraryTags[number]>[] = [
+	{
+		name: "DGS",
+		author: "Netflix",
+		gitHubRepo: "https://github.com/netflix/dgs-framework",
```

Comment:
All of these `gitHubRepo` fields need to remove the github.com link as those are already built in. Replace them with everything after the `.com/`

For example `gitHubRepo: "https://github.com/netflix/dgs-framework"` becomes `gitHubRepo: "netflix/dgs-framework"`